playpen/ansible/roles/qpidd/tasks/main.yml

```diff
@@ -2,10 +2,15 @@
 - name: Install packages
   dnf: name={{ item }} state=present
   with_items:
-    - qpid-cpp-server
-    - qpid-cpp-server-store
+    # qpid is broken in f23, so we have to install the compat packages:
+    #  https://bugzilla.redhat.com/show_bug.cgi?id=1286881
+    # - qpid-cpp-server
+    # - qpid-cpp-server-linerstore
+    - compat-qpid-cpp-server
+    - compat-qpid-cpp-server-store
```

Comment:
Do these packages exist on F22? If not, we may need a when statement to do this only for F23 and do the other for F22. Or not and we can force everyone to switch boxes.